meta-mender-demo/dynamic-recipes/mender-monitor/mender-monitor_%.bbappend

```diff
@@ -0,0 +1,8 @@
+
+
+do_install_append() {
+    oe_runmake \
+        -C ${S} \
+        DESTDIR=${D} \
+        enable-example-monitors
```

Comment:
Install and symlink the example monitors, not just simply install.

Reply:
By the way, is this comment not applicable anymore?

Reply:
Nope, the documentation instructs the user to enable the one he/she is testing manually, so that we can snipe out which alerts to show during demo.